libfuzzer-dotnet-windows.cc

```diff
@@ -134,11 +134,23 @@ FUZZ_EXPORT int __cdecl LLVMFuzzerInitialize(int *argc, char ***argv)
         die_sys("CreatePipe() failed");
     }
 
-    TCHAR sharedMemName[] = TEXT("LIBFUZZER_DOTNET_SHMEM");
```

Comment:
Would it be possible to avoid the loop by using a GUID for the shared memory name?